extensions/amp-carousel/0.1/base-slides.js

```diff
@@ -196,4 +208,21 @@ export class BaseSlides extends BaseCarousel {
       this.autoplayTimeoutId_ = null;
     }
   }
+
+  /**
+  * Remove autoplay.
+  * @protected
+  */
+  removeAutoplay() {
+    this.clearAutoplay();
+    this.hasAutoplay_ = false;
+    this.shouldAutoplay_ = this.hasAutoplay_ && this.isLoopingEligible();
+    if (this.element.hasAttribute('autoplay')) {
+      // Only remove if specified due to the `autoplay` attribute
+      this.element.removeAttribute('loop');
```

Comment:
this.hasLoop_ = false;
    this.shouldLoop = false;

too?

Reply:
Done